draft-ietf-mls-architecture.md

```diff
@@ -1579,16 +1579,16 @@ the following compromise scenarios:
 
 ### Compromise of Symmetric Keying Material {#symmetric-key-compromise}
 
-As described above, each MLS epoch creates a new Group Secret.
+As described above, each MLS epoch creates a new group Secret.
```

Comment:
The mixed caps here look funny to me. I would probably leave both capitalized, unless "Secret" is a term of art so used elsewhere.